src/jit/compiler.h

```diff
@@ -6868,9 +6868,24 @@ class Compiler
         bool vlIsOnStk(regNumber reg, signed offset);
     };
 
-    /*************************************************************************/
+private:
+
+    // Fill "varLoc" argument indicating the register position of the variable
+    // using "LclVarDsc" and "baseReg"/"offset" if it has a part in the stack (x64 bit float or long).
+    void fillRegisterVarLoc(LclVarDsc* varDsc, siVarLoc& varLoc, var_types type, regNumber baseReg, signed offset) const;
```

Comment:
If these stay in `Compiler` they should get some reasonable prefix, `si` or something.

Reply:
What is the reason of the prefix? Fast look up? I thought the 'gen' was for 'generate'.

Reply:
Personally I'd use a prefix simply because `Compiler` has a bazillion members, it makes it a bit easier to figure out where each function belongs.

Reply:
Yes, it is done for a fast look up. VS can't find references if they are under inactive `#ifdef` blocks for example.

Reply:
I believe the "si" of Scope Info is a good prefix for this. It is also called from genSetScopeInfo, so the "gen" would also make sense.